app/css/components/combo-button.css

```diff
@@ -16,14 +16,21 @@
         }
     }
 
+    & .--primary-segment.--disabled {
```

Comment:
@iHiD Could you check this? I needed to use a CSS class instead of the `disabled` property because the `disabled` property don't send out mouse events.